client/store/assessment/actions/postCountryStatus.ts

```diff
@@ -0,0 +1,27 @@
+import { createAsyncThunk } from '@reduxjs/toolkit'
+
+import axios from 'axios'
+import { ApiEndPoint } from '@common/api/endpoint'
+import { CountryIso } from '@meta/area'
+import { CountryStatus } from '@meta/assessment'
+
+export const postCountryStatus = createAsyncThunk<
+  CountryStatus,
+  {
+    countryStatus: CountryStatus
+    countryIso: CountryIso
+    name: string
+    cycleName: string
+    message?: string
+    notifyUsers?: boolean
+  }
+>('assessment/post/country/status', async ({ countryStatus, countryIso, name, cycleName, message, notifyUsers }) => {
+  const { data } = await axios.post(
+    ApiEndPoint.Assessment.countryStatus(countryIso, name, cycleName) + (notifyUsers ? '?notifyUsers=true' : ''),
+    {
+      countryStatus,
+      message,
```

Comment:
This is used for both '[ x ] Desk study' and any other countryStatus updates (ex. status change)
User notification (emails) are toggle with the url param (from function param)